### PR TITLE
🔧 fix(openidStrategy): avatar not persistent

### LINKED ISSUE #2056: [Bug]: Avatar with OIDC not persistent
### What happened?

Hello,

I used authentik to login via OIDC. When I set the avatar and logout and login again the avatar is gone.

### Steps to Reproduce

1. install librechat-dev, mongo and meilisearch container from scratch.
2. login via authentik
3. go to settings -> account -> avatar
4. choose file and upload
5. wait for confirmation
6. notice the avatar in the left-bottom corner
7. logout
8. login via authentik
9. gone

### What browsers are you seeing the problem on?

Vivaldi on Windows

### Relevant log output

sorry I don't see anything in the dev-librechat container log

### Screenshots

login:
image

choose image:
image

confirmation:
image

avatar:
image

logout and login again... gone:
image

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by berry-13**
## Summary

fixed #2056 

## Change Type

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing

tested with AWS cognito

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.
- [x] New documents have been locally validated with mkdocs

### CODE REVIEW

File: api/strategies/openidStrategy.js [hunk 1]
```diff
@@ -1,75 +1,40 @@
...
-      `[openidStrategy] downloadImage: Error downloading image at URL "${url}": ${error}`,
-    );
-    return '';
-  }
+const { handleExistingUser } = require('./process');
+
+const getFullName = (userinfo) => {
+  const { given_name, family_name, username, email } = userinfo;
+  return given_name && family_name
+    ? `${given_name} ${family_name}`
```
Comment (danny-avila) [inline api/strategies/openidStrategy.js]: Please don't nest ternary operators

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: api/strategies/openidStrategy.js (hunk 1 above)
Comment (berry-13) [reply api/strategies/openidStrategy.js]: is now cleaner?

**Comment by Berry-13** [comment]
want, don't merge this PR

**Comment by Berry-13** [comment]
closing this since we should prioritize a more robust solution by addressing the current shortcomings in handling the avatar within OpenID. The current implementation is not optimal and requires improvement

**Pull request closed without merging**